tachyon/zk/lookup/log_derivative_halo2/prover_impl.h

```diff
@@ -112,7 +112,7 @@ BlindedPolynomial<Poly, Evals> Prover<Poly, Evals>::ComputeMPoly(
   base::StableSort(storage.sorted_table_with_indices.begin(),
                    storage.sorted_table_with_indices.end());
 
-  std::vector<F> m_values(prover->pcs().N());
+  std::vector<F> m_values(prover->pcs().N(), F::Zero());
```

Comment:
It seems like loop below fills the values up to usable rows. just initialize the values for remaining indicies. But just add a comment that why this is safe.